### DISCUSSION TITLE
关于动漫季度识别问题(Regarding the identification of anime seasons)

### DISCUSSION THREAD

**Discussion by xiaojiejieouo** [jxxghp/MoviePilot#1270] [问题咨询]
![1](https://github.com/jxxghp/MoviePilot/assets/78160136/5bbe432c-bdcd-4064-bcc6-f2e8d4f162f6)
![2](https://github.com/jxxghp/MoviePilot/assets/78160136/24602d09-2893-4699-9757-e129bcf22bef)
![3](https://github.com/jxxghp/MoviePilot/assets/78160136/6827fc51-c9b1-4793-9854-ada71b158957)
![4](https://github.com/jxxghp/MoviePilot/assets/78160136/2b12f05a-dcda-457a-9959-90b3868a88c1)
![5](https://github.com/jxxghp/MoviePilot/assets/78160136/ba49550e-f69c-4c81-9080-4352eb5ee7ff)
如题,能够识别季度1,季度2,但是剧集名称全部是第一季的名称
As the question suggests, it is possible to identify quarter 1 and quarter 2, but all episode names are from the first season

**Comment by cng10001** [comment]
试试其他剧呢？

**Reply by xiaojiejieouo** [reply]
是的,我的bug体质叒触发了,在我这里至少百分之70以上的番剧都会只能识别第一季的名称
